public/js/graph/postsetup.js.jsx

```diff
@@ -0,0 +1,26 @@
+/*this was moved outside of setup.js*/
```

Comment:
I'm not certain that this comment adds anything useful to the file for future maintainers.

It is useful for reviewers of the PR, but I think that something like that should exist in either a description of what the PR does, or a line note.

By the way, why was it moved?